addons/mail/static/src/core/common/composer.xml

```diff
@@ -35,8 +35,8 @@
                 <div class="d-flex o-mail-Composer-bg flex-grow-1 border"
                     t-att-class="{
                         'o-mail-Composer-compactContainer m-1 shadow-sm border-secondary': compact and !props.composer.message,
-                        'o-mobile rounded-3': isMobileOS,
-                        'rounded-3' : normal,
+                        'o-mobile': isIOS,
```

Comment:
I know it was already there, but can't we just bootstrap margins? Or a class name that actually make sense? `o-mobile` applied only on IOS looks strange and does not convey the purpose of the class.

Reply:
Changed the class name to `o-ios-mobile`

Reply:
why not using the bootstrap class?

Reply:
Because we use the pseudo-class `:focus-within` in the scss using this class